From: Kouhei Maeda <mkouhei@palmtb.net>
Subject: Skip test_app_cleans_up_images
 (blockdiag.tests.test_command.TestBlockdiagApp)

Not worki test why I removes remote image url on diagram.
So skips the test case.
Last-Updated: 2021-01-31
---
 src/blockdiag/tests/test_command.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/blockdiag/tests/test_command.py b/src/blockdiag/tests/test_command.py
index f5e8e56..05269c8 100644
--- a/src/blockdiag/tests/test_command.py
+++ b/src/blockdiag/tests/test_command.py
@@ -16,11 +16,13 @@
 import os
 import unittest
 
+from nose.tools import nottest
 from blockdiag.command import BlockdiagApp
 from blockdiag.tests.utils import TemporaryDirectory
 
 
 class TestBlockdiagApp(unittest.TestCase):
+    @nottest
     def test_app_cleans_up_images(self):
         testdir = os.path.dirname(__file__)
         diagpath = os.path.join(testdir,
